submissions/2024-09-10-OpenFF-Iodine-Fragment-Opt-v1.0/README.md

```diff
@@ -0,0 +1,61 @@
+# OpenFF multi-Br ESP Fragment Conformers v1.0
+
+## Description
+
+A dataset containing fragments of molecules from the Zinc, enamine, and CHEMBL datasets, optimized at the B3LYP-D3BJ/DZVP level of theory. Molecules containing I were fragmented into small fragments. 
+Fragments containing I then had protomers enumerated.
+
+For each resulting molecule, a set of up to 5 conformers were generated by:
+  * generating a set of up to 1000 conformers with a RMS cutoff of 0.5 Å using the OpenEye backend of the OpenFF toolkit
+  * applying ELF conformer selection (max 5 conformers) using OpenEye
+
+Each conformer will be converged according to the 'GAU_LOOSE' criteria.
```

Comment:
```suggestion
```

Apologies I think I myself incorrectly copy-pasted this in an earlier submission -- this actually uses the default 'GAU' convergence criteria. I'll go correct this in a different PR

Reply:
Ah I thought I had caught these little readme typos--looks like I didn't actually commit them. Thank you for catching it!